src/redux/slices/areaSpecialization/middleware.ts

```diff
@@ -0,0 +1,30 @@
+import { IJobSeekerProps } from '@allTypes/reduxTypes/areaSpecializationTypes'
+import API from '@axios/API'
+import { IError } from '@axios/authentication/authManagerTypes'
+import { AppDispatch } from '@redux/store'
+
+import slice from './slice'
+
+const { setJobSeekerLoading, setError, setJobSeekerSubmitSuccess } = slice.actions
+
+const areaSpecialization = (params: IJobSeekerProps) => async (dispatch: AppDispatch) => {
+  try {
+    dispatch(setJobSeekerLoading(true))
+    await API.jobSeeker.jobSeeker(params)
+
+    dispatch(setJobSeekerSubmitSuccess(true))
+  } catch (error) {
+    dispatch(setError((error as IError).response?.data?.status.message))
+  } finally {
+    dispatch(setJobSeekerLoading(false))
```

Comment:
Namings are wrong, why this function name areaSpecialization but it's submitting jobSeeker form?

Reply:
done